packages/driver/src/cy/multi-domain/index.ts

```diff
@@ -149,7 +149,13 @@ export function addCommands (Commands, Cypress: Cypress.Cypress, cy: Cypress.cy,
         if (!communicator.listeners('uncaught:error').length) {
           communicator.once('uncaught:error', ({ err }) => {
             // @ts-ignore
-            Cypress.runner.onSpecError('error')({ error: err })
+            if (err.name === 'CypressError') {
```

Comment:
Do we think additional type safety might be beneficial here to handle any type of thrown error? These _should_ be reified by the communicator so this might not be a big deal 😅 
```suggestion
            if (err?.name === 'CypressError') {
```

Reply:
doesn't hurt